README.md

```diff
@@ -84,6 +77,7 @@ As the language is in its very early stages, every contribution will help in sha
 - [x] Default function arguments
 - [x] Panic function
 - [x] Varargs
+- [x] Standard library support for Linux

```

Comment:
The standard library isn't really a thing right now. It's in very very early stages (it doesn't even have a List/Vec or even a String struct) (Both will be added soon though).

I'd remove this

Reply:
We could add a `Standard library` section with things that we're planning on implementing or have already implemented. For example:
```
## Standard library

- [ ] OS Abstractions
- [ ] Formatting/Printing
- [ ] Filesystem API
- [ ] Networking and HTTP
- [ ] Date/Time
etc...
```